### PR TITLE
Update subctl to 0.13 in 2.6

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/23717

### CODE REVIEW

**Review by cadawson — APPROVED** [review]
Looks good to me. Thanks!

File: add-ons/submariner/submariner.adoc [hunk 1]
```diff
@@ -75,21 +75,21 @@ The `subctl` utility is shipped in a container image. Complete the following ste
 . Download the https://catalog.redhat.com/software/containers/rhacm2/subctl-rhel8/6229131e49e7196373df7d3e[`subctl` container] and extract a compressed version of the `subctl` binary to `/tmp` by entering the following command:
 +
 ----
-oc image extract registry.redhat.io/rhacm2/subctl-rhel8:v0.12 --path=/dist/subctl-v0.12.1-linux-amd64.tar.xz:/tmp/ --confirm
+oc image extract registry.redhat.io/rhacm2/subctl-rhel8:v0.13 --path=/dist/subctl-v0.13.1-linux-amd64.tar.xz:/tmp/ --confirm
```
Comment (Jaanki) [inline add-ons/submariner/submariner.adoc]:
```suggestion
oc image extract registry.redhat.io/rhacm2/subctl-rhel8:v0.13 --path=/dist/subctl-v0.13.0-linux-amd64.tar.xz:/tmp/ --confirm
```

The first version would be 0.13.0.

File: add-ons/submariner/submariner.adoc [hunk 2]
```diff
@@ -75,21 +75,21 @@ The `subctl` utility is shipped in a container image. Complete the following ste
...
 ----
 +
-*Note:* You might have to change `subctl-v0.12.1-linux-amd64.tar.xz` to the version of Submariner that you are using.  
+*Note:* You might have to change `subctl-v0.13.1-linux-amd64.tar.xz` to the version of Submariner that you are using.
```
Comment (Jaanki) [inline add-ons/submariner/submariner.adoc]:
```suggestion
*Note:* You might have to change `subctl-v0.13.0-linux-amd64.tar.xz` to the version of Submariner that you are using.  
```

**Review by Jaanki — CHANGES_REQUESTED** [review]
The first version would be 0.13.0.

**Comment by oafischer** [comment]
Thanks for pointing that out @Jaanki, went ahead and committed your edits. Thank you for your help on this issue! Going ahead with the merge now.

File: add-ons/submariner/submariner.adoc [hunk 3]
```diff
@@ -75,21 +75,21 @@ The `subctl` utility is shipped in a container image. Complete the following ste
...
 ----
 +
-*Note:* You might have to change `subctl-v0.12.1-linux-amd64.tar.xz` to the version of Submariner that you are using.  
+*Note:* You might have to change `subctl-v0.13.0-linux-amd64.tar.xz` to the version of Submariner that you are using.  
 
 . Decompress the `subctl` utility by entering the following command: 
 +
 ----
-tar -C /tmp/ -xf /tmp/subctl-v0.12-linux-amd64.tar.xz
+tar -C /tmp/ -xf /tmp/subctl-v0.13-linux-amd64.tar.xz
```
Comment (Jaanki) [inline add-ons/submariner/submariner.adoc]:
```suggestion
tar -C /tmp/ -xf /tmp/subctl-v0.13.0-linux-amd64.tar.xz
```

File: add-ons/submariner/submariner.adoc [hunk 4]
```diff
@@ -75,21 +75,21 @@ The `subctl` utility is shipped in a container image. Complete the following ste
...
 ----
 
 . Install the `subctl` utility by entering the following command:
 +
 ----
-install -m744 /tmp/subctl-v0.12/subctl-v0.12-linux-amd64 /$HOME/.local/bin/subctl
+install -m744 /tmp/subctl-v0.13/subctl-v0.13-linux-amd64 /$HOME/.local/bin/subctl
```
Comment (Jaanki) [inline add-ons/submariner/submariner.adoc]:
```suggestion
install -m744 /tmp/subctl-v0.13.0/subctl-v0.13.0-linux-amd64 /$HOME/.local/bin/subctl
```

**Review by Jaanki — CHANGES_REQUESTED** [review]
@oafischer Few more changes please.

**Comment by oafischer** [comment]
Done @Jaanki, thanks again.

**Pull request merged**